src/main/resources/assets/galacticraftcore/lang/de_DE.lang

```diff
@@ -414,11 +414,12 @@ gui.rocket.inv.name=Inventar / Treibstoff
 gui.rocket.warning.nogyroscope=Mysteriöse Kräfte sind im Spiel, die Gyroskope spielen verrückt!
 gui.rocket.warning.noparachute=Ich habe keinen Fallschirm! Wenn ich noch einmal Starten drücke, gibt es kein Zurück!
 gui.rocket.warning.nofuel=Ich brauche erst einmal etwas Raketentreibstoff!
-gui.chest.warning.wrongkey=Ich brauche wohl einen Dungeonschlüssel Stufe %s, um das zu entsperren!
+gui.chest.warning.wrongkey=Ich brauche wohl einen Dungeonschlüssel der Stufe %s, um das zu entsperren!
+gui.bedFail.message=Ich brauche eine kryogene Kammer, wenn ich im Weltraum schlafen möchte!
 gui.frequencymodule.warning0=Ich brauche wohl ein
```

Comment:
I looked at the English language file and it's missing there, too. Maybe it gets inserted later?